sia-react-native/screens/EventListFront.js

```diff
@@ -0,0 +1,189 @@
+import React, { Component } from 'react';
+import { FlatList, Text, Image, View, StyleSheet } from 'react-native';
+import ActionButton from 'react-native-action-button';
+import EventCard from "./EventCard";
+import { getEvents } from "../Backend";
+
+
+const styles = StyleSheet.create({
+    container: {
+        flex: 1,
+        alignItems: "center",
+        backgroundColor: '#15202A'
+    },
+    list: {
+        flex: 1,
+        paddingTop: 5
+    },
+
+    card: {
+        backgroundColor: '#fff',
+        flex: 1,
+        padding: 10,
+        paddingTop: 10,
+        paddingBottom: 20,
+        margin: 10,
+        marginTop: 5,
+        marginBottom: 5,
+    },
+
+    title: {
+        fontSize: 15,
+        fontWeight: '300',
+        marginLeft: 7,
+        textAlign: 'left',
+    },
+    title2: {
+        fontSize: 12,
+        fontWeight: '300',
+        marginLeft: 7,
+        textAlign: 'left',
+    },
+
+    title3: {
+        fontSize: 12,
+        fontWeight: '300',
+        marginLeft: 7,
+        textAlign: 'left',
+    },
+
+});
+
+
+
+class EventListFront extends Component {
+    state = {
+        events: [],
+    };
+
+    
+    componentDidMount() {
+           
+        this.props.navigation.addListener('didFocus', () => getEvents().then( events => this.setState({ events })))
```

Comment:
getEvents should be imported from the actions folder not the Backend file.

Reply:
this will cause an error

Reply:
Andre should fix these errors with his next pull request